static/elements/chromedash-featurelist.js

```diff
@@ -14,20 +14,23 @@ class ChromedashFeaturelist extends LitElement {
       metadataEl: {attribute: false}, // The metadata component element. Directly edited in template/features.html
       searchEl: {attribute: false}, // The search input element. Directly edited in template/features.html
       filtered: {attribute: false},
+      filteredWithState: {attribute: false},
     };
   }
 
   constructor() {
     super();
     this.features = [];
     this.filtered = [];
+    this.filteredWithState = [];
     this.metadataEl = document.querySelector('chromedash-metadata');
     this.searchEl = document.querySelector('.search input');
     this.whitelisted = false;
     this._hasInitialized = false; // Used to check initialization code.
     this._hasScrolledByUser = false; // Used to set the app header state.
     /* When scrollTo(), we also expand the feature. This is the id of the feature. */
     this._scrollOpenFeatureId = undefined;
+    this.openFeatures = new Map();
```

Comment:
nit: seems it's a id/boolean Map. use Set instead? `this.openedFeatures = new Set()`

Reply:
Done.